src/lib/fs-db.js

```diff
@@ -6,11 +6,18 @@ const chokidar = require('chokidar');
 const EventEmitter = require('events');
 const util = require('util');
 
+// Design note:
+//   "The default operation of the path module varies based on the operating system
+//   on which a Node.js application is running. Specifically, when running on a Windows
+//   operating system, the path module will assume that Windows-style paths are being used."
+// Therefore, to avoid inconsistent behaviors, we enforce working with POSIX paths at our
+// level.
+
 class FsDb {
   constructor(directory, logger) {
     EventEmitter.call(this);
 
-    this.directory = path.normalize(directory).replace(/\/$/, '');
+    this.directory = path.posix.normalize(directory).replace(/\/$/, '');
```

Comment:
That regexp was already here in the original code. This said, 'path' still leaves a trailing separator, if any, at the end of the configured directory in .env - There's no harm removing it to ensure the base directory does have any trailing separator afterwards.